README.md

```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
+
+Clone the `watson-voice-bot` repo locally. In a terminal, run:
+
+```
+git clone https://github.com/IBM/watson-voice-bot
+```
 
-1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy``.
+We’ll be using the file [`data/workspace.json`](data/workspace.json).
 
-2. In Toolchains, click on `Delivery Pipeline` to watch while the app is deployed. Once deployed, the app can be viewed by clicking `View app`.
+### 2. Create the services and deploy the web app
 
-   ![](doc/source/images/toolchain-pipeline.png)
+The next step is to deploy the application, we'll do this by running clicking the button below. A nice byproduct of doing doing this is that the services required (Speech-to-Text, Text-to-Speech, and Assistant) will be created automatically. Click the button below.
 
-3. To see the app and services created and configured for this Code Pattern, use the IBM Cloud dashboard. The app is named `CodePattern_VoiceBot-` with a unique suffix. The following services are created and easily identified by the `wvb-` prefix:
+[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-voice-bot.git)
+
+When the DevOps Toolchain Pipeline appears, click on the `Deploy` button to start the deployment. You can optionally, click on the `Delivery Pipeline` to watch the logs as the app is deployed.
+
+![](doc/source/images/toolchain-pipeline.png)
 
-   * wvb-speechtotext
-   * wvb-texttospeech
-   * wvb-assistant
+Once deployed, the app can be viewed by clicking `View app`. The app can also be viewed in the dashboard. The app should be prefixed with the string `watson-voice-bot-`, and the corresponding services that were created and easily identified by the `wvb-` prefix, i.e.: `wvb-assistant`.
 
-4. Import the Watson Assistant workspace. Follow Step [Import the Conversation workspace](#3-import-the-conversation-workspace) under the Section **Run Locally**
+### 3. Upload the Watson Assistant workspace
 
-   ![](doc/source/images/application_name.png)
+Now that our services are created and the app is deployed we need to update tell the application to use a specific Watson Assistant dialog. We'll be using the file [`data/workspace.json`](data/workspace.json), which documents our entire conversation dialog. To do this, launch the Watson Assistant tool and use the `import` icon button on the right. Find the [`data/workspace.json`](data/workspace.json) file from the cloned repo and import that to the Watson Assistant tool.
 
-5. Configure Environment Variable. Open the app starting with the name `CodePattern_VoiceBot` from under the [IBM Cloud Dashboard](https://console.bluemix.net/dashboard/apps/) by clicking on the Application name.
+Each workspace in Watson Assistant has a specific ID, to find the `Workspace ID` for a given workspace, click the context menu of the workspace and select `View details`. The workspace ID can be copied and saved as we'll need it in the next step.
```

Comment:
Think this needs a period in place of the first comma.

Reply:
done